packages/preview/isc-hei-report/0.1.0/template/report.typ

```diff
@@ -0,0 +1,223 @@
+#import "@preview/modern-isc-report:0.1.0" : *
```

Comment:
There are still a few places where it says `modern-isc-report` (also in the README). Here, in particular, it prevents the template from working. Once changed here, it works on my system.

Reply:
This should now be resolved. Sorry for the mess, everything has now been double checked and should be working as expected.